snooty/parser.py

```diff
@@ -424,6 +436,54 @@ def dispatch_departure(self, node: docutils.nodes.Node) -> None:
                 target.children = [identifier]
                 item.term.append(target)
 
+    def handle_tabset(self, node: n.Directive, line: int) -> None:
+        tabset = node.options["tabset"]
+        # retrieve dictionary associated with this specific tabset
+        tab_definitions_list = specparser.SPEC.tabs[tabset]
```

Comment:
We should throw an error here and terminate gracefully if the parser encounters a tabset not included in the `rstspec.toml`.